### ISSUE TITLE
[A11y] Incorrect role attribute on breadcrumb headings

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#9310]
Issue: 
The breadcrumb heading `<h1>` tags on pages like the 'Report Package' page cannot have the role "list". These serve more as headings anyway, so no role attribute should be required.

https://www.nuget.org/packages/Newtonsoft.Json/13.0.2-beta2/ReportAbuse
